openedx/core/djangoapps/user_api/views.py

```diff
@@ -449,11 +449,12 @@ def _add_level_of_education_field(self, form_desc, required=True):
         # form used to select the user's highest completed level of education.
         education_level_label = _(u"Highest level of education completed")
 
+        options = [(name, _(label)) for name, label in UserProfile.LEVEL_OF_EDUCATION_CHOICES]  # pylint: disable=translation-of-non-string
```

Comment:
@antoviaque That's correct, translation would happen to early if it was done at `UserProfile` class load time. What kind of comment did you have in mind? Something like *"The options are marked for translation in UserProfile model definition."*?